### DISCUSSION TITLE
Cassandra support?

### DISCUSSION THREAD

**Discussion by aozmen121** [micronaut-projects/micronaut-data#1546] [Ideas]
Will there be a future support for Cassandra to be included as part of micronaut-data?

**Comment by dstepanov** [comment]
We do plan to support more NoSQL databases, next will probably be cloud-native ones.

**Reply by aozmen121** [reply]
Thank you for the prompt update, would you be able to give me a time scale as to when and which cloud-native DBs will be supported in the future versions of micronaut-data?

**Reply by dstepanov** [reply]
Maybe @graemerocher will know more about the time.

Looking at the Cassandra driver it shouldn't be a complicated task, we already support a lot of things that are needed to implement the repository operations: SQL-like query builder, tabular structure to entity mapper, sync and reactive support, etc.

**Reply by graemerocher** [reply]
It is doable, but probably not in the short term unless we receive a community contribution. Support for Cloud document databases and Graph databases is higher at this point in time.
